storybook/config/preview.tsx

```diff
@@ -43,7 +43,7 @@ export const parameters = {
         'Components',
         ['Buttons', 'Containers', 'Feedback', 'Forms', 'Layout', 'Media', 'Navigation', 'Text'],
         'Pages',
-        ['Introduction', 'Amsterdam.nl', ['Home']],
+        ['Introduction', 'Amsterdam.nl', ['Home Page'], 'Amopis'],
```

Comment:
Should we generalize to ‘Website’ and ‘Application’ instead of ‘Amsterdam.nl’ and ‘Amopis’?

Reply:
I agree, Amopis is just one team. And we're looking for a general internal application theme.